mappings/net/minecraft/container/MerchantContainer.mapping

```diff
@@ -1,22 +1,31 @@
 CLASS ayy net/minecraft/container/MerchantContainer
-	FIELD c villager Lbgd;
-	FIELD d villagerInventory Layx;
+	FIELD c trader Lbgd;
+	FIELD d traderInventory Layx;
 	FIELD e levelProgress I
 	FIELD f canLevel Z
 	METHOD <init> (ILavq;)V
 		ARG 1 syncId
+		ARG 2 playerInventory
 	METHOD <init> (ILavq;Lbgd;)V
 		ARG 1 syncId
+		ARG 2 playerInventory
+		ARG 3 trader
 	METHOD a setRecipes (Lbgf;)V
 	METHOD a setCanLevel (Z)V
 		ARG 1 canLevel
 	METHOD b equals (Lbbu;Lbbu;)Z
 		ARG 1 itemStack
 		ARG 2 otherItemStack
+	METHOD c pullMatchingStackFromPlayerInventory (ILbbu;)V
```

Comment:
Why not just `takeFromPlayer`?

Reply:
It is autofill, like what recipe books do.

Reply:
But isn't there a shorter name? At least `Inventory` is unnecessary.